Web_Development/FrontEnd/CSS Padding/CSS Padding.md

```diff
@@ -0,0 +1,101 @@
+# CSS Padding
+It can be used for creating space outside the element .There are properties for setting the padding for each side of an element (top, right, bottom, and left).
+## When to use this?
+When we need to create space outside a container or element to adjust it's position.

```

Comment:
Since this is the beginning of the file the content should not be this short. Add 2-3 more lines in the same. The rest looks okayish.